conda-forge.yml

```diff
@@ -4,3 +4,4 @@ travis:
 appveyor:
   secure:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
+max_py_ver: '37'
```

Comment:
@megies this is the "official way" to get py37 but bare in mind that the bot is doing that in topological order, so this PR may need to wait for packages upstream.